Database/src/main/resources/changelog1.xml

```diff
@@ -811,4 +811,31 @@
         </insert>
     </changeSet>
 
+    <changeSet author="CAF" id="1520597520212-1">
+        <createTable tableName="tbl_global_config">
```

Comment:
Can we discuss the name of this table?  I think it should be something like `tbl_tenant_settings`.

Reply:
After discussion we've decided to keep the current name and will have a further Jira to introduce a `scope` column which will clarify that the configuration setting is per-tenant.